src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/GeneralPartnerDataDto.java

```diff
@@ -37,10 +40,12 @@ public class GeneralPartnerDataDto {
     private LocalDate dateOfBirth;
 
     @JsonProperty("nationality1")
-    private String nationality1;
+    @ValidEnum(message = "Nationality must be valid")
+    private Nationality nationality1;
 
     @JsonProperty("nationality2")
-    private String nationality2;
+    @ValidEnum(message = "Nationality must be valid")
```

Comment:
If validation for second nationality is being implemented, I'd have thought message should be "Second nationality must be valid".